src/library/scala/Option.scala

```diff
@@ -323,6 +323,43 @@ sealed abstract class Option[+A] extends Product with Serializable {
   final def zip[A1 >: A, B](that: Option[B]): Option[(A1, B)] =
     if (isEmpty || that.isEmpty) None else Some((this.get, that.get))
 
+  /** Converts an Option of a pair into an Option of the first element and an Option of the second element.
```

Comment:
This is a little wordy, but I guess it is just a copy of the doc for `IterableOps`?

Reply:
That is partially correct - copied from Array

Reply:
It's too bad `Option` doesn't implement `AbstractIterable` or some such and just inherit its documentation for free.

Reply:
I think it should implement Traversable, but `++` is hard for Option to implement since Option only contains one element.

Reply:
I think it's better not to add more dependencies of `Option` (a small, nearly self-contained thing) onto the collections API (a big, complicated thing).

(If anything, I might prefer to go in the other direction, deprecate the implicit, and require an explicit method call to make an `Iterable` or `Traversable` from an `Option`.)

To really talk about it, it might be better to move to https://contributors.scala-lang.org. We (the contributors community) have gone around about this before, so we'd probably want to start by digging up past discussions (pretty sure there was at least one in scala/bug, perhaps elsewhere as well).

Reply:
Well, `Traversable` is deprecated last I checked, but my point is that there is some commonality between collections and Option, so why not have a trait represent it.

Yeah, I was also going to suggest that this was probably brought up in the strawman repo, or in the merge, or at least I hope ti was discussed.